package.json

```diff
@@ -21,7 +21,7 @@
   },
   "main": "electron/main.mjs",
   "dependencies": {
-    "@gptscript-ai/gptscript": "^0.9.5-rc3",
+    "@gptscript-ai/gptscript": "github:thedadams/node-gptscript#de914911013bf9da02046906b88a5613dce508e1",
```

Comment:
Same here

Reply:
I wonder how this worked because that commit doesn't exist for that repo.